### DISCUSSION TITLE
How do I disable the feed?

### DISCUSSION THREAD

**Discussion by riastradh** [community/community#58470] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

The feed is useless.  How do I opt out of it?

**Comment by the-officialjosh** [comment]
😪 I share that sentiment too but I guess it is here to stay as dozens of issues have been raised over the past weeks complaining of this new UI

**Comment by meesterdude** [comment]
I also would love to be able to disable this.
